src/services/scope/uc/scope.facade.js

```diff
@@ -0,0 +1,42 @@
+const ScopeUc = require('./scope.uc');
+
+class ScopeBcFacade {
```

Comment:
class FacadeInterface 

class ScopeBcFacade extends FacadeInterface 
that also hold or ref the facades and can use to locate it this.facade(<path>)
-force setup
- check by registration if instanceof adding class is a facade
-do not or add the app to enforce that app.service can not be used
but if we add the app we should do in in the setup with this.app = app;